pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -3535,14 +3535,12 @@
     },
     "System.Numerics.Tensors": {
       "StableVersions": [
-        "0.1.0",
-        "0.3.0"
+        "0.1.0"
       ],
-      "BaselineVersion": "0.3.0",
+      "BaselineVersion": "0.1.0",
       "InboxOn": {},
       "AssemblyVersionInPackageVersion": {
-        "0.1.0.0": "0.1.0",
-        "0.2.0.0": "0.2.0"
+        "0.1.0.0": "0.1.0"
```

Comment:
We need an entry for the `0.2.0` package right?

Reply:
we will need that if we ever ship that 0.2.0 version of the package